packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -392,6 +397,10 @@ export class QuoteRequestor {
 
         const typedMakerUrls = standardUrls.concat(altUrls);
 
+        const cancelTokenSource = axios.CancelToken.source();
```

Comment:
are there any issues with using the same `cancelToken` across many requests?

Reply:
Excellent question - I was concerned about this myself. 

So I ended up testing this in a toy repo: https://github.com/phil-ociraptor/playground/blob/main/index.js

This functions exactly as if we had a cancel token per request, which was what I did the first time: https://github.com/phil-ociraptor/playground/commit/2aa544dd7870a02996539e3a54eabab1389e1919#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346